docs/cryoet_data_portal_docsite_quick_start.md

```diff
@@ -9,6 +9,16 @@ This page provides details to start using the CryoET Data Portal.
 
 ## Installation
 
+### Requirements
+
+The CryoET Data Portal Client requires a Linux or MacOS system with:
+
+- Python 3.7 to Python 3.10. Or R, supported versions TBD.
```

Comment:
I believe we should be supporting python 3.11 as well. Also, the client currently is only for python. We don't support R.

Reply:
Updated.